app/client/cli/event.go

```diff
@@ -51,9 +51,7 @@ func (c *Client) CmdEventCreate() *cobra.Command {
 		Example: `hkn event create esboot -name "ES Bootcamp" -a 5 -c 30 -e scan,sql,hb -f kali`,
 		Args:    cobra.MinimumNArgs(1),
 		Run: func(cmd *cobra.Command, args []string) {
-			ctx, cancel := context.WithTimeout(context.Background(), 3*time.Minute)
-			defer cancel()
-
+			ctx := context.Background()
```

Comment:
This completely removes any timing out, do we want that?

Reply:
It will be canceled out when VMs are started, when all VM's are properly started, the channel will be closed so there will be EOF, then it will automatically closed.

Reply:
What if the daemon hangs indefinitely (I am playing the devil's advocate here 😉)? Should the client also hang indefinitely? I am fine with changing it, but I am just making sure that you know of the implication of changing this.

Reply:
However, if some VMs are NOT initialised correctly in given timeout period, then we might have other issues as well. If there is a problem on Daemon side, it will also cancel out on client side.

Reply:
👍